docker-build/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.3'
+services:
+  sql_service:
+    build:
+      context: .
+      dockerfile: Dockerfile 
+    image: sql_service
+    restart: unless-stopped
+    depends_on: 
+     - db
+    stdin_open: true
+    networks:
+      - db-network
+  db:
+    image: mariadb:10.4
+    restart : always
+    ports:
```

Comment:
I ended up removing this port assignment because it was having some problems on my computer. It might have just been an issue with 3306 being used for some other service on my end.